<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainerPokemon</class>
 <widget class="QWidget" name="TrainerPokemon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Ability Index:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>EV:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="7">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="line_move_1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="line_move_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="line_move_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="line_move_4"/>
       </item>
      </layout>
     </item>
     <item row="10" column="1" colspan="7">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="spinBox_7"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_8"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_9"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_10"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_11"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_12"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Item</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>IV:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Happiness:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="5">
      <widget class="QLineEdit" name="line_name"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Species:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Ball:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Nature:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="8">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Form:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_form_2"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Level:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_level"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Gender:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_gender">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>Male</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Female</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Unknown</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="check_shiny">
         <property name="text">
          <string>Shiny</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="check_super_shiny">
         <property name="text">
          <string>Super Shiny</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="check_shadow">
         <property name="text">
          <string>Shadow</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Moves:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_species"/>
     </item>
     <item row="9" column="1" colspan="7">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="spinBox"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_3"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_5"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_6"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_4"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_2"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="1" colspan="7">
      <widget class="QLineEdit" name="line_item"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
